x/ibc/testing/chain.go

```diff
@@ -158,18 +173,17 @@ func (chain *TestChain) QueryProof(key []byte) ([]byte, uint64) {
 // CONTRACT: this function must only be called after app.Commit() occurs
 func (chain *TestChain) NextBlock() {
 	// set the last header to the current header
-	chain.LastHeader = ibctmtypes.CreateTestHeader(
-		chain.CurrentHeader.ChainID,
-		chain.CurrentHeader.Height,
-		chain.CurrentHeader.Time,
-		chain.Vals, chain.Signers,
-	)
+	chain.LastHeader = chain.CreateTMClientHeader()
 
 	// increment the current header
 	chain.CurrentHeader = abci.Header{
-		Height: chain.CurrentHeader.Height + 1,
-		Time:   chain.CurrentHeader.Time,
+		Height:  chain.App.LastBlockHeight() + 1,
+		AppHash: chain.App.LastCommitID().Hash,
+		Time:    chain.CurrentHeader.Time,
```

Comment:
Is time not getting increased at all?

Reply:
Yeah I’d add 7 seconds

Reply:
it is, but the coordinator. Increasing time here would cause updating clients to use headers from the future